docs/self-managed/setup/deploy/openshift/redhat-openshift.md

```diff
@@ -12,9 +12,9 @@ We test against the following OpenShift versions and guarantee compatibility wit
 
 | OpenShift version | Supported          |
 | ----------------- | ------------------ |
-| 4.11.x            | :white_check_mark: |
```

Comment:
Not sure if that's needed since OpenShift 4.11 officially [reached end-of-life](https://docs.openshift.com/dedicated/osd_architecture/osd_policy/osd-life-cycle.html#sd-life-cycle-dates_osd-life-cycle) on Dec 10, 2023 (by Red Hat).